spec/std/char_spec.cr

```diff
@@ -326,7 +326,7 @@ describe "Char" do
     it { 'a'.in_set?("ab", "ac", "ad").should be_true }
 
     it "rejects invalid ranges" do
-      expect_raises do
+      expect_raises ArgumentError, "Invalid range c-a" do
```

Comment:
Similar to the other changes, please use `()` around method call when a block is used. Thank you.

Reply:
Ah yes, I started doing this somewhere in between, but later forgot to edit the prior ones...

Reply:
Hm, there are 218 occassions in the stdlib specs ...

Reply:
> Hm, there are 218 occassions in the stdlib specs

Leave those for another PR, let's focus this PR only on the ones modified by you 😉 

Thank you!

Reply:
Well, I was already at it... and it was nearly easier to take all at once